Content.Server/GameTicking/GameTicker.GameRule.cs

```diff
@@ -47,14 +90,18 @@ public bool HasGameRule(string rule)
 
         public void ClearGameRules()
         {
-            foreach (var rule in _gameRules.ToArray())
+            _addedGameRules.Clear();
```

Comment:
```suggestion
```
`EndGameRule(rule)` checks only ends rules that are in `_addedGameRules` and will already remove them.

Also, should this be looping over `_startedGameRules` or `_addedGameRules`?